.ci/aarch64/filter_docker_aarch64.sh

```diff
@@ -33,6 +35,10 @@ filter_and_build()
 
 main()
 {
+	# install yq if not exist
+        if ! command -v yq >/dev/null; then
+		install_yq
+	fi
```

Comment:
Could you fix the indentation here - you seen to have three levels of it :)

btw, you could simplify to something like the following:

```
[ -z "$(command -v yq)" ] && install_yq
```

Reply:
right. prefer the simplification. 😁